docs/common/error-page-template.md

```diff
@@ -0,0 +1,22 @@
+# Error Code: {module}-{code}
+**Applies to:** {system}
+
+---
+
+## Message
+
+> {message}
+
+## Cause
+
+{description}
+
+### Solution
```

Comment:
Is this supposed to be an h3?

Reply:
the solution specifically, dunno why it selected four rows, I'm on mobile

Reply:
No, good catch :+1: